src/typed-method-types/workflows/triggers/inputs.ts

```diff
@@ -11,9 +11,17 @@ export type InputParameterSchema = {
 };
 
 /** The structure that must be provided to the workflow input to pass a value */
-type WorkflowInput = {
+type WorkflowInput = WorkflowInputValue | WorkflowInputCustomizableValue;
+
+type WorkflowInputValue = {
   // deno-lint-ignore no-explicit-any
   value: any;
+  customizable_value?: never;
+};
+
+type WorkflowInputCustomizableValue = {
+  customizable_value: boolean;
+  value?: never;
```

Comment:
We don't want to have `false` as an option either I'm assuming since that wouldn't make sense considering we would need `value` to be set instead then right?  I think we can keep `WorkflowInputValue` to have this type:
```
type WorkflowInputValue = {
  // deno-lint-ignore no-explicit-any
  value: any;
  customizable_value?: never;
};
```
What do you think?

Reply:
Ah, I assumed it'd default to `false` on the BE! Keeping `WorkflowInputValue` using `never` seems good to me